akamai/resource_akamai_property_activation.go

```diff
@@ -174,12 +174,22 @@ func resourcePropertyActivationExists(d *schema.ResourceData, meta interface{})
 
 	network := papi.NetworkValue(d.Get("network").(string))
 	version := d.Get("version").(int)
+
+	/*_, ok := d.GetOk("version")
+	if !ok {
+		la, _ := activations.GetLatestActivation(papi.NetworkValue(strings.ToUpper(d.Get("network").(string))), papi.StatusActive)
```

Comment:
This looks like it was to handle the case of version being optional and/or the case of "latest" being incremented to handle changes in rule structure .  But it is commented out.  How do we handle looking up latest active value when version is missing. Was this not needed or part of another change deemed not yet ready for release?

Reply:
Provider supports supplying version in TF file or optional/latest does provoke an update to state file (update in place) with no changes to server side behaves like a refresh operation

Reply:
I know what the current code does.  I was asking 
1) how we handle activate with latest when other operations like property ensureEditableVersion can change what version number is latest
2) was this commented code from debugging work or is there more development going on here I might not have been aware of?
Your reply seems to miss some key words and punctuation and is hard to follow as a result.

Reply:
1) Activate when driven without parameter version relies on 
if version, ok := d.GetOk("version"); ok && version.(int) != 0 {
		activation.PropertyVersion = version.(int)
	} else {
		version, err := property.GetLatestVersion("", correlationid) <-- (this calls ensureEditableVersion() so matches property resource which calls that in CREATE/UPDATE first)
2) This was debug code I have removed in latest commit